### DISCUSSION TITLE
AutoBuilder 1.0.0-pre2

### DISCUSSION THREAD

**Discussion by adaliea** [adaliea/AutoBuilder#23] [Announcements]
Changes since the end of the 2022 season
- Add support for the @AutoBuilderAccessible Annotation
- Fully implement text highlighting in text boxes
- Squash a _ton_ of bugs
- Remove the use of JavaFX, reducing the jar size and making the jar work on linux and macos
- Many more user interface improvements
- Migrate to using the Robot Side library 

See our wiki [here](https://github.com/FRC3476/AutoBuilder/wiki) to learn how to use this app!

<hr /><em>This discussion was created from the release <a href='https://github.com/FRC3476/AutoBuilder/releases/tag/1.0-pre2'>AutoBuilder 1.0.0-pre2</a>.</em>

**Comment by ghost** [comment]
The Auto Builder website is a comprehensive resource for automotive enthusiasts covering a wide range of topics such as street rods, hot rods, late model cars, drag racing, handling, compact cars, street and offroad trucks, and performance boats. It also covers various automotive parts and systems like chassis, engine, electrical systems, exhaust, fuel system, transmission, suspension, steering, brakes, wheels and tires, interior, and exterior accessories. Additionally, the site offers sections on engine and interior swaps, notable builders in the automotive industry, and original equipment manufacturers (OEMs).
https://theautobuilder.com/the-grand-national-roadster-show-2024/

**Comment by ghost** [comment]


**Reply by ghost** [reply]
https://theautobuilder.com/the-grand-national-roadster-show-2024/
